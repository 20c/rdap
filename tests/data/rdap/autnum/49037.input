{
  "handle": "AS49037",
  "startAutnum": 49037,
  "endAutnum": 49037,
  "name": "PG19",
  "status": [
    "active"
  ],
  "entities": [
    {
      "handle": "ORG-PRL15-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "PROSTIE RESHENIA LLC"
          ],
          [
            "kind",
            {},
            "text",
            "org"
          ],
          [
            "adr",
            {
              "label": "ST. MOVSESA HORENACI, 14/20\n1109\nVAGARSHAPAT\nARMENIA"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+37444120474"
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+37441057581"
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "mikhail.majorov@gmail.com"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/49037",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/ORG-PRL15-RIPE"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "RIPE-NCC-END-MNT",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "RIPE-NCC-END-MNT"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ],
          [
            "org",
            {},
            "text",
            "ORG-NCC1-RIPE"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/49037",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/RIPE-NCC-END-MNT"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "WA2477-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "WEBROCKET SUPPORT TEAM"
          ],
          [
            "kind",
            {},
            "text",
            "group"
          ],
          [
            "adr",
            {
              "label": "st. Movsesa Horenaci 14/20, Vagharshapat, Armavir, Armenia, 1101"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "ripe@webrocket.am"
          ],
          [
            "email",
            {
              "type": "abuse"
            },
            "text",
            "abuse@webrocket.am"
          ]
        ]
      ],
      "roles": [
        "technical",
        "administrative"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/49037",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/WA2477-RIPE"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "WEBROCKET-MNT",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "WEBROCKET-MNT"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/49037",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/WEBROCKET-MNT"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "WA2477-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "WEBROCKET SUPPORT TEAM"
          ],
          [
            "kind",
            {},
            "text",
            "group"
          ],
          [
            "adr",
            {
              "label": "st. Movsesa Horenaci 14/20, Vagharshapat, Armavir, Armenia, 1101"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "ripe@webrocket.am"
          ],
          [
            "email",
            {
              "type": "abuse"
            },
            "text",
            "abuse@webrocket.am"
          ]
        ]
      ],
      "roles": [
        "abuse"
      ],
      "entities": [
        {
          "handle": "WEBROCKET-MNT",
          "vcardArray": [
            "vcard",
            [
              [
                "version",
                {},
                "text",
                "4.0"
              ],
              [
                "fn",
                {},
                "text",
                "WEBROCKET-MNT"
              ],
              [
                "kind",
                {},
                "text",
                "individual"
              ]
            ]
          ],
          "roles": [
            "registrant"
          ],
          "links": [
            {
              "value": "https://rdap.db.ripe.net/autnum/49037",
              "rel": "self",
              "href": "https://rdap.db.ripe.net/entity/WEBROCKET-MNT"
            },
            {
              "value": "http://www.ripe.net/data-tools/support/documentation/terms",
              "rel": "copyright",
              "href": "http://www.ripe.net/data-tools/support/documentation/terms"
            }
          ],
          "objectClassName": "entity"
        }
      ],
      "objectClassName": "entity"
    }
  ],
  "remarks": [
    {
      "description": [
        "****************** Routing Policy *******************",
        "********** DI-DI",
        "********** DOM.RU",
        "********** MEDIASETI",
        "********** RETN",
        "********** LEVEL-MSK",
        "********** RND-IX",
        "********** SEA-IX",
        "********** VK.COM",
        "********** MEGAFON",
        "********** Eurasia Peering",
        "********** APPLE",
        "********** GOOGLE",
        "********** MAIL.RU",
        "********** Hurricane Electric",
        "********** MICROSOFT",
        "********** OVH",
        "********** TWITCH",
        "********** FACEBOOK",
        "********** MSK-IX",
        "********** PITER-IX MSK",
        "********** PITER-IX HEL",
        "********** PITER-IX RIX",
        "********** PITER-IX SPB",
        "********** PITER-IX TIL",
        "********** PITER-IX UA",
        "********** GBL",
        "***************************************************************",
        "***** BGP communities in advertisements towards downstream ****",
        "49037:55001 Taganrog",
        "49037:55002 Rostov-on-Don",
        "49037:55003 Krasnodar",
        "49037:55004 Moscow",
        "*********** BGP communities accepted from downstream **********",
        "Change of RR list's happens by handmade via e-mail to noc@pg19.ru"
      ]
    }
  ],
  "links": [
    {
      "value": "https://rdap.db.ripe.net/autnum/49037",
      "rel": "self",
      "href": "https://rdap.db.ripe.net/autnum/49037"
    },
    {
      "value": "http://www.ripe.net/data-tools/support/documentation/terms",
      "rel": "copyright",
      "href": "http://www.ripe.net/data-tools/support/documentation/terms"
    }
  ],
  "events": [
    {
      "eventAction": "registration",
      "eventDate": "2009-03-31T07:49:08Z"
    },
    {
      "eventAction": "last changed",
      "eventDate": "2023-07-04T07:07:32Z"
    }
  ],
  "rdapConformance": [
    "nro_rdap_profile_asn_flat_0",
    "cidr0",
    "rdap_level_0",
    "nro_rdap_profile_0"
  ],
  "notices": [
    {
      "title": "Filtered",
      "description": [
        "This output has been filtered."
      ]
    },
    {
      "title": "Whois Inaccuracy Reporting",
      "description": [
        "If you see inaccuracies in the results, please visit:"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/49037",
          "rel": "inaccuracy-report",
          "href": "https://www.ripe.net/contact-form?topic=ripe_dbm&show_form=true",
          "type": "text/html"
        }
      ]
    },
    {
      "title": "Source",
      "description": [
        "Objects returned came from source",
        "RIPE"
      ]
    },
    {
      "title": "Terms and Conditions",
      "description": [
        "This is the RIPE Database query service. The objects are in RDAP format."
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/49037",
          "rel": "terms-of-service",
          "href": "http://www.ripe.net/db/support/db-terms-conditions.pdf",
          "type": "application/pdf"
        }
      ]
    }
  ],
  "port43": "whois.ripe.net",
  "objectClassName": "autnum"
}