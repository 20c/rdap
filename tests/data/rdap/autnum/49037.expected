{"org_address": "", "org_name": "", "emails": ["netadm@tagan.ru"], "name": "MGL-AS"}
